clusters/build-clusters/02_cluster/README.md

```diff
@@ -0,0 +1,73 @@
+# 01-Cluster
+
+[02-Cluster](https://console-openshift-console.apps.build02.gcp.ci.openshift.org) is an OpenShift-cluster managed by DPTP-team. It is one of the clusters for running Prow job pods.
+
+## Installation
+
+The gcp project `openshift-ci-build-farm` for installation of this cluster: [gcp console](https://console.cloud.google.com/home/dashboard?project=openshift-ci-build-farm). The project is created by [DPP team](hhttps://issues.redhat.com/browse/DPP-4926).
+
+We created the public zone (base domain for installer) and the service account `ocp-cluster-installer`.
+Since this project is dedicated for build farm. We granted the `Owner` role to this account.
+
+Get the pull secret by `oc get secret -n ci cluster-secrets-gcp -o jsonpath='{.data.pull-secret}' | base64 -d`.
+
+Generate `install-config.yaml` (uploaded to BW already for reuse): 
+
+```
+./openshift-install create install-config
+? SSH Public Key /Users/hongkliu/.ssh/id_rsa_build02.pub
+? Platform gcp
+? Service Account (absolute path to file or JSON content)
+/Users/hongkliu/Downloads/build02.install/openshift-ci-build-farm-64e4ce412ae3.json
+INFO Saving the credentials to "/Users/hongkliu/.gcp/osServiceAccount.json"
+? Project ID openshift-ci-build-farm
+? Region us-east1
+INFO Credentials loaded from file "/Users/hongkliu/.gcp/osServiceAccount.json"
+? Base Domain gcp.ci.openshift.org
+? Cluster Name build02
+? Pull Secret
+```
+
+Customize machine type in the `install-config.yaml`:
```

Comment:
Its not clear to me what `Customize machine type` means. Do you have a sed? Is there anything that prevents us from just storing what we need in the `install-config.yaml` rather than requiring to change it?

Reply:
Add the doc link for it.
I do not have `sed` for this. But the download version should be good enough to understand (together with the doc).